docs/user_guide/hopsfs/xattrs.rst

```diff
@@ -0,0 +1,86 @@
+===========================
+Extended Attributes in HopsFS
+===========================
+
+Overview
+--------------------
+
+*Extended attributes* (abbreviated as *xattrs*) are a filesystem feature that allow user applications to associate additional metadata with a file or directory. Unlike system-level inode metadata such as file permissions or modification time, extended attributes are not interpreted by the system and are instead used by applications to store additional information about an inode. Extended attributes could be used, for instance, to specify the character encoding of a plain-text document.
+
+HopsFS extended attributes
+~~~~~~~~~~~~~~~~~~~~~~~
+Extended attributes in HopsFS are modeled after extended attributes in Linux (see the Linux manpage for `attr(5) <http://www.bestbits.at/acl/man/man5/attr.txt>`_ and `related documentation <http://www.bestbits.at/acl/>`_. An extended attribute is a *name-value pair*, with a string name and binary value. Xattrs names must also be prefixed with a *namespace*. For example, an xattr named *myXattr* in the *user* namespace would be specified as **user.myXattr**. Multiple xattrs can be associated with a single inode.
+
+Namespaces and Permissions
+~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
@maismail the length of the headline needs to match the length of the tilts, otherwise a warning is thrown.